proteus/mprans/SW2DCV.h

```diff
@@ -1034,243 +911,191 @@ class SW2DCV : public SW2DCV_base {
       // CALCULATE EDGE BASED CFL //
       //////////////////////////////
       double mi = lumped_mass_matrix[i];
-      edge_based_cfl[i] = 2 * fabs(dLowii) / mi;
+      edge_based_cfl[i] = 1.0 * fabs(dLowii) / mi;
```

Comment:
was this a bug or was one of the other factors modified?

Reply:
No, not a bug. This just changes the CFL condition by dividing it by 2. It's not a big deal. I only changed it because the TAMU code has the factor of 1 and I like the codes to be "exactly" the same when I compare them.